weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -736,7 +737,75 @@ def obj_create(self, req: tsi.ObjCreateReq) -> tsi.ObjCreateRes:
             data=[list(ch_obj.model_dump().values())],
             column_names=list(ch_obj.model_fields.keys()),
         )
-        return tsi.ObjCreateRes(digest=digest)
+
+        return tsi.ObjCreateRes(
+            digest=digest,
+            object_id=req.obj.object_id,
+        )
+
+    @ddtrace.tracer.wrap(name="clickhouse_trace_server_batched.create_obj_batch")
+    def obj_create_batch(self, req: tsi.ObjCreateBatchReq) -> tsi.ObjCreateBatchRes:
```

Comment:
because this is internal, I think using an interface class doesn't really make sense. Can we just parameterize it? We use the request objects for actual requests but in this case we only use it internally in clickhouse batched right?